Userland/Libraries/LibC/resolv.h

```diff
@@ -6,11 +6,22 @@
 
 #pragma once
 
+#include <netinet/in.h>
 #include <sys/cdefs.h>
 #include <sys/types.h>
 
 __BEGIN_DECLS
 
-int res_query(char const* dname, int class_, int type, unsigned char* answer, int anslen);
+struct __res_state {
+    char* dnsrch[1];
+    char defdname[256];
+    int nscount;
+    struct sockaddr_in nsaddr_list[1];
+};
+
+typedef struct __res_state* res_state;
+
+int res_query(char const* dname, int class_, int type, unsigned char* answer,
+    int anslen);
```

Comment:
this should just be on the same line, artifact of your mixed clang-format configs.

Reply:
Thanks, fixed with the next push.

Reply:
With the `dnslookup` feature disabled, it doesn't even need the additions in `resolv.h`. Should I remove them?

Reply:
I would say that the definitions you've added look less like a feature and more like stubs, so if there's any that don't need to be added, let's not add them for this PR. it does make me wonder if we have an issue open for actually implementing all this IPv6 stuff...